index.html

```diff
@@ -1 +1,12 @@
-<div>Hello</div>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Give the title a better name like the project name you want it to be.
Just a suggestion not compulsory in this case but remember from next time,